solutions/pytest_folder/auth_keys_stepik.json

```
@@ -0,0 +1,4 @@
+{
+  "login": "example@example.ex",
+  "password": "example"
+}
```

[Причины](https://stackoverflow.com/a/729795).

```suggestion
}

```